manager/assets/modext/widgets/element/modx.tree.element.js

```diff
@@ -563,6 +563,32 @@ Ext.extend(MODx.tree.Element,MODx.tree.Tree,{
             });
         }
 
+        if (n.isLoaded()) {
+            var childNodes = n.childNodes;
```

Comment:
Hmmm...
`const childNodes = n.childNodes;`?

Reply:
Hmmm....
`let childNodes = n.comeBackToCourses;`

Reply:
Hmmmmmmm...
If a childNodes is never reassigned, using the `const` is better. 

> let childNodes = n.comeBackToCourses;

Ha. Ha. Ha.